init.cmd

```diff
@@ -0,0 +1,4 @@
+@echo off
```

Comment:
❗️  This file is not necessary, and impairs usability by undermining the parameter help support provided by PowerShell. This repository uses PowerShell directly.

Reply:
Removed